css-paint-api/background-image-alpha-ref.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#background {
+  background-color: yellow;
+  display: inline-block;
+}
+
+.container {
+  font-size: 0px;
+}
+</style>
+</head>
+<body>
+
+<div id="background">
+  <div class="container">
+    <canvas id ="opaque" width="100" height="100"></canvas>
+  </div>
+  <div class="container">
+    <canvas id ="nonopaque" width="100" height="100"></canvas>
+  </div>
+</div>
+
+<script>
+function drawCanvas(canvasID, hasAlpha) {
```

Comment:
This seems a bit elaborate for the reference file, would divs with `box-sizing: border-box; border: 4px solid blue` achieve the same result?